### PR TITLE
Wiring Nightwatch into Circle

### PR DESCRIPTION
**Pull request by timwright12**
Adding nightwatch tests into Circle to run daily at 4pm

Tests passing in Circle:
https://app.circleci.com/pipelines/github/department-of-veterans-affairs/content-build/785/workflows/a7582185-bb28-421d-9ad5-b843d1887dd8

tests are now in a daily job in this PR

:hat_tip: to @tskinn for the help here

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ services:
     image: content-build:${IMAGE_TAG:-latest}
     build:
       context: .
+      target: base
     volumes:
       - .:/application
```
Comment (tskinn) [inline docker-compose.yml:9]: I think with the new Dockerfile changes this needs to be updated
```suggestion
      - .:/application/content-build
```

File: Dockerfile [hunk 2]
```diff
@@ -32,10 +33,29 @@ RUN aws --version # Verify AWS CLI installation.
 # Explicitly set CA cert to resolve SSL issues with AWS.
 ENV AWS_CA_BUNDLE /aws/dist/botocore/cacert.pem
 
-RUN mkdir -p /application
+RUN mkdir -p /application/content-build
+RUN chown -R vets-website:vets-website /application
 
-WORKDIR /application
+WORKDIR /application/
```
Comment (tskinn) [inline Dockerfile:39]:
```suggestion
WORKDIR /application/content-build
```

I think if we leave this out the normal docker-compose will not function as people expect. I'm not sure

File: script/run-docker-nightwatch.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-node src/platform/testing/e2e/test-server.js --buildtype=vagovstaging --host=0.0.0.0 &
+node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --host=0.0.0.0 &
```
Comment (billfienberg) [inline script/run-docker-nightwatch.sh:3]: I have no objection to using `vagovprod` over `vagovstaging`, but I am curious about the reason for this change.

**Review by billfienberg — APPROVED** [review]
Tests passing + ops helping with Docker changes = LGTM

File: script/run-docker-nightwatch.sh (hunk 3 above)
Comment (timwright12) [reply script/run-docker-nightwatch.sh:3]: Just so we're running tests against a production version of the site

**Pull request merged**